src/view/View.js

```diff
@@ -679,6 +679,38 @@ var View = Base.extend(Emitter, /** @lends View# */{
         this._matrix.append(matrix);
     },
 
+    /**
+     * Transform the view so that its {@link #bounds} fit within the
+     * specified rectangle.
+     *
+     * @param {Rectangle} rectangle
+     * @param {Boolean} [fill=false]
+     *
+     * @example {@paperscript height=100}
+     * // Fitting the view to the bounding rectangle of an item's bounding:
+     *
+     * // Create a circle at the top left corner of the view.
+     * var item = new Path.Circle({
+     *    center: [0, 0],
+     *    radius: 50,
+     *    fillColor: 'orange'
+     * });
+     *
+     * // Make view fit its bounds.
+     * view.fitBounds(item.bounds);
+     */
+    fitBounds: function(rectangle, fill) {
+        rectangle = Rectangle.read(arguments);
+        var bounds = this.getBounds();
+        var horizontalRatio = bounds.width / rectangle.width;
+        var verticalRatio = bounds.height / rectangle.height;
+        var scale = fill
+            ? Math.max(horizontalRatio, verticalRatio)
+            : Math.min(horizontalRatio, verticalRatio);
+        this.translate(bounds.center.subtract(rectangle.center));
+        this.scale(scale);
```

Comment:
@sasensi this is the same code as on Item, yes? Can't we reuse it and safe some space, e.g. through a function on `Rectangle`?

Reply:
@lehni, actually, I wrote a different code than `item.fitBounds()` because `View` didn't have a `setBounds()` method and the item version is relying on this method.
So if you want to reuse the code, we also need to find a way to share the `item.setBounds()` code with the `View` class.
Maybe making `view.bounds` settable makes sense too ?